cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/IndexedTable.java

```diff
@@ -100,7 +100,7 @@
   /**
    * Column key used to store the existence of a row in the secondary index.
    */
-  private static final byte[] IDX_COL = {'r'};
+  public static final byte[] IDX_COL = {'r'};
```

Comment:
does it have to be public? Since we are in the API, I am not sure we want to make this public.